src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -253,15 +254,24 @@ namespace winrt::TerminalApp::implementation
         auto weakThis{ get_weak() };
 
         // Handle it on a subsequent pass of the UI thread.
-        co_await winrt::resume_foreground(Dispatcher(), CoreDispatcherPriority::Normal);
-        if (auto page{ weakThis.get() })
+        for (const auto& action : _startupActions)
         {
-            for (const auto& action : _startupActions)
+            co_await winrt::resume_foreground(Dispatcher(), CoreDispatcherPriority::Normal);
```

Comment:
is it bad that we're changing from "move to UI thread before all commands" to "move to UI thread during every command"? Does this put us back into the weird world of unreliabel commandline dispatch for the windows+r case?

Reply:
I just spawned a whole _bunch_ from win+r and they all seemed fine to me. I can't remember off the top of my head if this was actually important for this changelist or not, I'll see if I can revert it.

Reply:
It sure _wasn't_ necessary